### PR TITLE
Metadata docs

### PR DESCRIPTION
**Pull request by noahmacaskill**
Some general corrections and improvements.

The SD card capacity I've extracted is in KiB, could convert to GB (or GiB), looking at the Reclog file it is in GiB. Changed bits per sample to bits per second (bits per sample as I understand it is bit depth, which is already there, bits per second was missing). Also removed byte rate and block align since they aren't part of our final metadata output.

A couple other small improvements.

### CODE REVIEW

**Comment by atruskie** [comment]
Nice. 

I'd prefer to normalize both of those memory card values to bytes and bytes per second if possible - better than changing the docs to be inconsistent.

File: src/Emu/Metadata/FrontierLabs/LogFileExtractor.cs [hunk 1]
```diff
@@ -11,6 +11,9 @@ namespace Emu.Metadata.FrontierLabs
 
     public class LogFileExtractor : IMetadataOperation
     {
+        public const int MegabyteConversion = 1048576;
+        public const int KilobyteConversion = 1024;
```
Comment (atruskie) [inline src/Emu/Metadata/FrontierLabs/LogFileExtractor.cs:15]: These are incorrect?. Normally these are the correct values to use, but the units you're converting are already measured as <x>bytes.  e.g. these would be correct converting from decimal units to bytes.

I double checked, so I think I'm correct here... the numbers in Fixtures.yaml don't make sense

image

Whereas, simply scaling magnitudes:

image

image

Comment (atruskie) [reply src/Emu/Metadata/FrontierLabs/LogFileExtractor.cs:15]: It should be noted that SD uses MB/s (the base 10 scale) everywhere as opposed to MiB/s (the base 2 scale): https://www.sdcard.org/developers/sd-standard-overview/bus-speed-default-speed-high-speed-uhs-sd-express/

I'm fairly sure I'm correct....

**Review by atruskie — CHANGES_REQUESTED** [review]

**Comment by noahmacaskill** [comment]
Sorry it took so long on this, had my hands tied on some other work. I'm realizing the capacity values were so far off because of overflow, need a long to store that # of bytes. Aside from that, I think you're right, the capacity and speed values are in base 10.

**Pull request merged**